bin/pygrb/pycbc_pygrb_pp_workflow

```diff
@@ -251,9 +251,10 @@ for snr_type in timeseries:
         timeseries_plots = _workflow.FileList([])
         # Plots without and with injections
         for inj_file in set([None, tuning_inj_file]):
-            # If the inj_file is used for the plot, include the tuning
-            # injection set tag in assembling the plot name
-            tags = [snr_type] if inj_file is None else [snr_type, tuning_inj_set]
+            # If the tuning injection file is used for the plot, include its
+            # tags in assembling the plot name
+            tags = [snr_type] if inj_file is None \
+                else [snr_type] + inj_file.tags
```

Comment:
It feels like there's a bunch of places in this loop where you have this if/else structure. Why not do:
```
inj_tags = inj_file.tags if inj_file else []
```
and then `tags = [snr_type] + inj_tags` here, and similar in all places below.